engine/src/platform.h

```diff
@@ -487,6 +487,7 @@ enum MCPlatformSystemProperty
 	kMCPlatformSystemPropertyCursorImageSupport,
     
     kMCPlatformSystemPropertyVolume,
+	kMCPlatformSystemPropertySystemAppearance,
```

Comment:
Group the properties by kind - so separate this from the previous one as its from a logical distinct area.